### ISSUE TITLE
mention import of doobie.implicits.javatime._ in documentation

### ISSUE THREAD

**Issue by poohsen** [tpolecat/doobie#1397]
I started out with the documentation for custom mappings which mentioned support for Java's OffsetDateTime and ZonedDateTime out of the box but I had to go to stackoverflow to find the right import for these implicits (`import doobie.implicits.javatime._`)

The FAQ's mention `import doobie.implicits.javatimedrivernative._` but a) this is only available since 0.12 and b) this hint shouldn't only be in the FAQs anyway.

**Comment by jatcwang** [comment]
Thanks for the feedback @poohsen. `import doobie.implicits.javatime._` is deprecated which is why it was removed from the docs. The reason being it's actually not the right import for many databases because it assumes native support from the driver for those types. (e.g. iirc MySQL and Postgres driver doesn't support java.time.Instant natively, so it breaks if you try to use it).

I agree that since we use time types all the time it should be featured more prominently, but I'm not sure if "Custom Mappings" is the right place for it. Got any suggestions?

**Comment by poohsen** [comment]
0.12 worked for me. 
If the custom mappings page is not the right one then I think it could be mentioned under "postgres extensions" that the implicits now feature the trait `JavaTimeInstances` which covers OffsetDateTime, etc. Right now it only mentions Instant and Timestamp.

**Comment by jatcwang** [comment]
Yes you're right, I didn't know that section exist (https://tpolecat.github.io/doobie/docs/15-Extensions-PostgreSQL.html#java-8-time-types-jsr310-). Thanks for the info.

Would you like to submit a PR to help fix it? :)
